trinity/utils/chains.py

```diff
@@ -94,6 +95,32 @@ def get_database_socket_path(data_dir: Path) -> Path:
     ))
 
 
+DATABASE_DIR_NAME = 'chain'
+
+
+def get_database_base_dir(data_dir: Path) -> Path:
+    return data_dir / DATABASE_DIR_NAME
+
+
+def get_database_dir(database_base_dir: Path, sync_mode: str) -> Path:
+    if sync_mode == SYNC_FULL:
+        return database_base_dir / 'full'
+    elif sync_mode == SYNC_LIGHT:
+        return database_base_dir / 'light'
+    else:
+        raise ValueError(
+            f"Unknown sync mode: `{sync_mode}`, must be one of "
+            f"{SYNC_FULL}/{SYNC_LIGHT}"
+        )
+
+
+DATABASE_ENGINE_LOCK_NAME = '.trinity.db_engine.lock'
```

Comment:
I see what you're saying with `lock`, but also I'm used to lock files being things that should go away after shutdown.

Maybe?
- `.trinity.db_engine.freeze`
- `.trinity.db_engine.frozen`
- `.trinity.db_engine.initialized`
- `.trinity.db_engine.selected`

(and all the other places that use `lock` for this choice)

Reply:
went with `marker`